test/e2e/Deploy.t.sol

```diff
@@ -163,18 +414,35 @@ contract E2EDeploymentMainnetTest is DeployMainnet, CommonDeploymentTest {
     uint256 _updateRateDelay = pidRateSetter.params().updateRateDelay;
     skip(_updateRateDelay);
 
+    vm.mockCall(
+      OP_CHAINLINK_ETH_USD_FEED,
+      abi.encodeWithSelector(IChainlinkOracle.latestRoundData.selector),
+      abi.encode(uint80(1), uint256(2000e8), uint256(0), uint256(block.timestamp - 1), uint64(0))
+    );
```

Comment:
The same `mockCall()` is being repeated thrice. I think just one would be enough.

Also, why do you need to mock it?

Reply:
the difference is that `block.timestamp` is different at each call, we need to mock it because when we skip time, the chainlink feed is now older, hence invalid 🤦